packages/definitions/src/fieldType.ts

```diff
@@ -196,3 +212,61 @@ export function convertDBTypeToFieldType(
 
   throw new Error(`No matching found for dialect ${dbDialect} and type ${dbType}`)
 }
+
+export type LocoKitFieldRule =
+  | {
+      fieldId: string
+      operator: '$eq'
+      value: string | number
+    }
+  | {
+      fieldId: string
+      operator: '$in'
+      value: string[] | number[]
+    }
+
+export type LocoKitField = {
```

Comment:
This name seems ambiguous to me. Because it is related to **form** fields, but LocoKit also has **table** fields in its database schema. Both are pretty interlinked, so is this type `LocoKitField` assumed to join these two notions in a single type?

If not, I think we should rename it `LocoKitFormField` and apply a similar renaming to the related types and variables.

Reply:
You're right, I update the code.